core/AppSpec.js

```diff
@@ -39,8 +39,7 @@ export class AppSpec {
      *      app has loaded and fully initialized, including elapsed time of asset loading and init.
      * @param {boolean} [c.webSocketsEnabled] - true to enable Hoist websocket connectivity,
      *      establish a connection and initiate a heartbeat..
-     * @param {boolean} [c.idleDetectionEnabled] - true to enable auto-suspension by `IdleService`.
-     * @param {Class} [c.idleDialogClass] - Component or element factory used to indicate App has
```

Comment:
if we are actually going to change this, should probably just be `idlePanel` as it is no longer specifically a Class.